packages/components/src/Disclosure/Disclosure.css

```diff
@@ -24,15 +28,18 @@
 }
 
 .summary:hover,
-.summary:focus {
-  background: var(--color-surface--hover);
+.summary:focus-visible {
```

Comment:
Focus visible is how we have been treating interactions across all focusable components by default. Disclosure is a little older and was missed during the retheme.